nilearn/connectome/rena_clustering.py

```diff
@@ -0,0 +1,646 @@
+
+    Parameters
+    ----------
+    mask : filename, niimg, NiftiMasker instance, optional default None)
+        Mask to be used on data. If an instance of masker is passed,
+        then its mask will be used. If no mask is it will be computed
+        automatically by a NiftiMasker.
+
+    n_clusters : int, optional (default 2)
+        The number of clusters to find.
+
+    scaling : bool, optional (default False)
+
+    memory : instance of joblib.Memory or string
+        Used to cache the masking process.
+        By default, no caching is done. If a string is given, it is the
+        path to the caching directory.
+
+    memory_level : integer, optional (default 1)
+        Rough estimator of the amount of memory used by caching. Higher value
+        means more memory for caching.
+
+    verbose : int, optional (default 1)
+        Verbosity level.
+
+    smoothing_fwhm : float, optional
+        If smoothing_fwhm is not None, it gives the size in millimeters of the
+        spatial smoothing to apply to the signal.
+
+    standardize : boolean, optional
+        If standardize is True, the time-series are centered and normed:
+        their variance is put to 1 in the time dimension.
+
+    target_affine : 3x3 or 4x4 matrix, optional (default None)
+        This parameter is passed to image.resample_img. An important use-case
+        of this parameter is for downsampling the input data to a coarser
+        resolution (to speed of the model fit). Please see the related
+        documentation for details.
+
+    target_shape : 3-tuple of integers, optional (default None)
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass : None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass : None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r : float, optional (default None)
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details.
+
+    n_iter : int, optional (default 10)
+        Number of iterations of the recursive neighbor agglomeration
+
+    threshold : float in the open interval (0., 1.), optional (default 1e-7)
+        Threshold used to handle eccentricities.
+
+    Attributes
+    ----------
+    `masker_` : instance of NiftiMasker
+        The nifti masker used to mask the data.
+
+    `mask_img_` : Nifti like image
+        The mask of the data. If no mask was supplied by the user,
+        this attribute is the mask image computed automatically from the
+        data `X`.
+
+    `labels_ ` : array-like, (n_features,)
+        cluster labels for each feature.
+
+    `n_clusters_` : int
+        Number of clusters.
+
+    `sizes_` : array-like (n_features,)
+        It contains the size of each cluster.
+    """
```

Comment:
A suggestion here: in the builder signature, group together the parameters that are specific to ReNA on one hand, and those that are common to all maskers on the other hand.